#!/usr/bin/python3
def raise_exception():
    error_code = 5 * "40"
